endpoints/meeting-search.php

```diff
@@ -26,10 +34,14 @@
     $text_space = " ";
     $message = "";
 
-function sendSms($message)
+function sendSms($message, $is_mobile)
 {
     if (isset($_REQUEST['From']) && isset($_REQUEST['To'])) {
-        $GLOBALS['twilioClient']->messages->create($_REQUEST['From'], array("from" => $_REQUEST['To'], "body" => $message));
+        if (has_setting('mobile_check') && json_decode(setting('mobile_check')) && ($is_mobile == true)) {
```

Comment:
using a function would simplify things here.  then you can also clean up the function by not passing around paramters